build.sbt

```diff
@@ -88,3 +89,180 @@ lazy val core = project
+        "1.2.0",
+        "*"
+      ).mkString(java.io.File.separator),
+      Seq(
+        s"${csrCacheDirectory.value}",
+        "https",
+        "repo1.maven.org",
+        "maven2",
+        "org",
+        "glassfish",
+        "jaxb",
+        "txw2",
+        "2.2.11",
+        "*"
+      ).mkString(java.io.File.separator),
+      Seq(
+        s"${csrCacheDirectory.value}",
+        "https",
+        "repo1.maven.org",
+        "maven2",
+        "relaxngDatatype",
+        "relaxngDatatype",
+        "20020414",
+        "*"
+      ).mkString(java.io.File.separator),
+      Seq(
+        s"${csrCacheDirectory.value}",
+        "https",
+        "repo1.maven.org",
+        "maven2",
+        "com",
+        "sun",
+        "xsom",
+        "xsom",
+        "20140925",
+        "*"
+      ).mkString(java.io.File.separator),
+      Seq(
+        s"${csrCacheDirectory.value}",
+        "https",
+        "repo1.maven.org",
+        "maven2",
+        "com",
+        "sun",
+        "xml",
+        "bind",
+        "external",
+        "rngom",
+        "2.2.11",
+        "*"
+      ).mkString(java.io.File.separator),
+      Seq(
+        s"${csrCacheDirectory.value}",
+        "https",
+        "repo1.maven.org",
+        "maven2",
+        "com",
+        "sun",
+        "istack",
+        "istack-commons-runtime",
+        "2.21",
+        "*"
+      ).mkString(java.io.File.separator),
+      Seq(
+        s"${csrCacheDirectory.value}",
+        "https",
+        "repo1.maven.org",
+        "maven2",
+        "org",
+        "glassfish",
+        "jaxb",
+        "codemodel",
+        "2.2.11",
+        "*"
+      ).mkString(java.io.File.separator),
+      Seq(
+        s"${csrCacheDirectory.value}",
+        "https",
+        "repo1.maven.org",
+        "maven2",
+        "org",
+        "glassfish",
+        "jaxb",
+        "jaxb-core",
+        "2.2.11",
+        "*"
+      ).mkString(java.io.File.separator),
+      Seq(
+        s"${csrCacheDirectory.value}",
+        "https",
+        "repo1.maven.org",
+        "maven2",
+        "javax",
+        "xml",
+        "bind",
+        "jaxb-api",
+        "2.1",
+        "*"
+      ).mkString(java.io.File.separator),
+      Seq(
+        s"${csrCacheDirectory.value}",
+        "https",
+        "repo1.maven.org",
+        "maven2",
+        "com",
+        "sun",
+        "istack",
+        "istack-commons-tools",
+        "2.21",
+        "*"
+      ).mkString(java.io.File.separator),
+      Seq(
+        s"${csrCacheDirectory.value}",
+        "https",
+        "repo1.maven.org",
+        "maven2",
+        "org",
+        "glassfish",
+        "jaxb",
+        "jaxb-xjc",
+        "2.2.11",
+        "*"
+      ).mkString(java.io.File.separator),
+      Seq(
+        s"${csrCacheDirectory.value}",
+        "https",
+        "repo1.maven.org",
+        "maven2",
+        "com",
+        "sun",
+        "xml",
+        "bind",
+        "jaxb-impl",
+        "2.2.11",
+        "*"
```

Comment:
Is adding all these paths manually really the expected usage of this? This seems incredibly fragile and hard to maintain, especially with the verisions in the. Isn't any version change going to break this? Does the sbt xjc plugin not handle all of this?

Reply:
I'm not thrilled with this either, but I don't know another way of doing it. This is here because the sbt-xjc plugin invokes java from the command line, and it needs to find the libraries downloaded as dependencies during the build somehow. These lines are adding the required jars to the classpath.

Reply:
> find the libraries downloaded as dependencies

These are dependencies of xjc correct? Seems odd that this would be needed, I would expect xjc to handle that. I wonder if we are using/configuring it incorrectly? None of the examples in the xjc repo require manually listing xjc dependencies.

Reply:
I also would expect it to handle this automatically, but it was throwing ClassNotFoundExceptions before I added all of this. Maybe there is some extra configuration needed, but nothing jumps out at me from their documentation.

@arosien is there a smarter way to have the java process recognize/find the jars?

Reply:
It looks like sbt-xjc is already adding jars to the classpath:

https://github.com/sbt/sbt-xjc/blob/master/src/main/scala/com/github/retronym/sbtxjc/SbtXjcPlugin.scala#L96

I do see references to Ivy in that file, it's possible it doesn't work with Coursier? You might try settting `useCoursier := false`, in your sbt configs? We do that for daffodil because something doesn't work with coursier.

Reply:
I added that setting, commented out the classpath lines I added, and I'm getting the ClassNotFoundExceptions. It looks like it's re-downloading the dependencies, so I'm assuming that means it's not using couriser anymore. I'll dig around with this for a bit to see if I can get it to work.